run/README.tslist

```diff
@@ -15,15 +16,26 @@ Bogus point A             pt_a   29.718  -75.772
 Bogus point B             pt_b   37.614  -74.650
 
 
+If cell locations are to be used (such as for idealized cases), the i/j 
+locations are to be specified as follows:
+
+#-----------------------------------------------#
+# 24 characters for name | pfx |   J   |    I   |
```

Comment:
Did we agree that this was going to be lat lon, but also i j (not J I)?

Reply:
The header should be I, J (not J, I). Should I update the README.tslist file?

Reply:
Yes, please update this README.

Reply:
See attached file.
[README.tslist.txt](https://github.com/wrf-model/WRF/files/2815237/README.tslist.txt)

Reply:
I was confused with the attachment of a text file. I just modified the file README.tslist.

Reply:
I've never used github - modifying and attaching a file seemed easiest.

Reply:
@phawbeck 
OK, gotcha.